site/_quarto.yml

```diff
@@ -42,6 +42,8 @@ website:
           file: developer/get-started-developer-framework.qmd
         - text: "{{< fa cubes >}} Supported Models"
           file: developer/model-documentation/supported-models.qmd
+        - text: "{{< fa book-medical >}} Generating Model Documentation"
```

Comment:
I don't recall why we ever used title-style caps for this dropdown menu (maybe because the navbar uses them? 🤔), but perhaps this would be a good time to switch to sentence-style caps since we're referencing actual topic titles?

Reply:
Yeah, I just followed what was already there — I kind of like it this way because like you say the navbar has them, and this makes it consistent, but up to you...